ocs_ci/utility/version.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf8 -*-
+"""
+Module for version related util functions.
+"""
+
+from semantic_version import Version
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import defaults, ocp
+
+
+def get_semantic_version(version, only_major_minor=False, ignore_pre_release=False):
+    """
+    Returning OCP semantic version from config.
+
+    Args:
+        version (str): String version (e.g. 4.6)
+        only_major_minor (bool): If True, only major and minor will be parsed.
+        ignore_pre_release (bool): If True, the pre release version will be ignored
+
+    Retruns
+       semantic_version.base.Version: Object of semantic version for OCP.
+
+    """
+    version = Version.coerce(version)
+    if only_major_minor:
+        version.patch = None
+        version.prerelease = None
+    elif ignore_pre_release:
+        version.prerelease = None
+    return version
+
+
+# Version constants
+VERSION_4_2 = get_semantic_version("4.2", True)
```

Comment:
can we move constants to top of the file?

Reply:
These are initialized using `get_semantic_version` which is defined above.

Reply:
I wanted, but as Coady said, it's not possible as it's utilizing the function which has to be defined before.